scylla/src/transport/session_test.rs

```diff
@@ -1181,6 +1181,78 @@ async fn test_timestamp() {
     assert_eq!(results, expected_results);
 }
 
+#[ignore = "works on remote Scylla instances only (local ones are too fast)"]
+#[tokio::test]
+async fn test_request_timeout() {
+    use std::time::Duration;
+    let uri = std::env::var("SCYLLA_URI").unwrap_or_else(|_| "127.0.0.1:9042".to_string());
+
+    {
+        let session = SessionBuilder::new()
+            .known_node(uri.as_str())
+            .build()
+            .await
+            .unwrap();
+        session
+            .await_timed_schema_agreement(Duration::from_secs(1))
+            .await
+            .unwrap();
```

Comment:
Nit: waiting for schema agreement explicitly is no longer required, by default the driver now waits for schema agreement after every schema-altering query.